systems/controllers/controlUtil.cpp

```diff
@@ -419,11 +420,81 @@ Vector6d bodyMotionPD(RigidBodyManipulator *r, DrakeRobotState &robot_state, con
   des_rpy = body_pose_des.tail<3>();
   angleDiff(pose_rpy,des_rpy,error_rpy);
   body_error.tail(3) = error_rpy;
-
+  
   Vector6d body_vdot = (Kp.array()*body_error.array()).matrix() + (Kd.array()*(body_v_des-J*robot_state.qd).array()).matrix() + body_vdot_des;
   return body_vdot;
 }
 
+Vector6d bodySpatialMotionPD(RigidBodyManipulator *r, DrakeRobotState &robot_state, const int body_index, const Vector6d &body_xyzexp_des, const Vector6d &body_v_des, const Vector6d &body_vdot_des, const Vector6d &Kp, const Vector6d &Kd, const Isometry3d &T_task_to_world)

```

Comment:
Why are we passing in the desired pose as a Vector6d with exponential coordinates instead of using an Isometry3d?